### PR TITLE
INS-3539 kill backupmanager test + some minor changes for INS-3562

### PR DESCRIPTION
**Pull request by afiskon**

### CODE REVIEW

File: run-test.py [hunk 1]
```diff
@@ -751,9 +755,41 @@ def test_stop_start_heavy(heavy_pod, pod_ips, restore_from_backup = False):
...
+    start_test("test_kill_backupmanager" + ("_restore_from_backup" if restore_from_backup else ""))
+    info("==== kill backupmanager "+("with restore from backup " if restore_from_backup else "")+"test started ====")
+    alive = wait_until_insolar_is_alive(pod_ips, NODES, step="before-killing-backupmanager")
+    check_alive(alive)
+
+    info("Running benchmark and trying to kill backupmanager on pod #"+str(heavy_pod))
+    ssh(heavy_pod, "tmux new-session -d -s backupmanager-killer " +
+        """\\"while true; do killall -9 -r backupmanager; sleep 0.1; done""" +
+        """; bash\\" """)
+    ok = run_benchmark(pod_ips, extra_args='-s --members-file=' + MEMBERS_FILE, r=100, timeout=100)
```
Comment (megge-dream) [inline run-test.py]: Why save members to file, if you don't use it after?

File: run-test.py [hunk 2]
```diff
@@ -413,28 +413,32 @@ def get_finalized_pulse_from_exporter():
...
+    try:
+        out = ssh_output(ssh_pod, 'cd go/src/github.com/insolar/insolar && ' +
+                         'timelimit -s9 -t'+str(timeout)+' ' +
+                         './bin/benchmark -c ' + str(c) + ' -r ' + str(r) + ' -a http://'+pod_ips[virtual_pod_name] +
+                         ':'+str(port) + '/admin-api/rpc ' +
+                         ' -p http://'+pod_ips[virtual_pod_name]+':'+str(port + 100)+'/api/rpc ' +
+                         '-k=./scripts/insolard/configs/ ' + extra_args)
+    except Exception as e:
+        print(e)
+        out = "ssh_output() throwed an exception (non-zero return code)"
```
Comment (megge-dream) [inline run-test.py]: What about added strerr from failed command to this message?

Comment (afiskon) [reply run-test.py]: OK, sure.

File: run-test.py (hunk 1 above)
Comment (afiskon) [reply run-test.py]: OK, I'll fix this.

**Pull request merged**